OtripleS.Web.Api/Services/StudentExamFees/StudentExamFeeService.Validations.cs

```diff
@@ -12,30 +12,30 @@ namespace OtripleS.Web.Api.Services.StudentExamFees
 {
     public partial class StudentExamFeeService
     {
-        private void ValidateStudentExamFeeId(Guid studentExamFeeId)
-        {
-            if (studentExamFeeId == default)
-            {
-                throw new InvalidStudentExamFeeException(
-                    parameterName: nameof(StudentExamFee.Id),
-                    parameterValue: studentExamFeeId);
-            }
-        }
+        //private void ValidateStudentExamFeeId(Guid studentExamFeeId)
+        //{
+        //    if (studentExamFeeId == default)
+        //    {
+        //        throw new InvalidStudentExamFeeException(
+        //            parameterName: nameof(StudentExamFee.Id),
+        //            parameterValue: studentExamFeeId);
+        //    }
+        //}
```

Comment:
@viralpandya go through your code and make sure you don't have any commented code in your PRs please.

Reply:
fixed this one but not in this PR. Removing it at 4 pm

Reply:
removed